app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
+
```

Comment:
- [ ] Also there, we should have something like this:

````ruby
class Post < ApplicationRecord
  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
  has_many :likes
  has_many :comments

  def five_most_recent_comments
    comments.order(created_at: :desc).limit(5)
  end

  after_save :update_posts_counter

  private

  def update_posts_counter
    author.update(posts_counter: author.posts.count)
  end

  
end

````

Reply:
Hello @joel-kalema, thank you so much for your nice review and giving great feedback I am going to add.
Thank 👍